content/_index.html

```diff
@@ -70,3 +70,57 @@ <h1>What is OpenCue?</h1>
 {{% /blocks/feature %}}
 
 {{< /blocks/section >}}
+
+<div class="body-container whats-new">
+	<div class="container">
+		<div class="row">
+			<div class="col-lg-12 center-text" style="margin-bottom: 40px">
+				<h2 class="section-title">What's New</h2>
```

Comment:
So we'll have to update this section whenever new notes are added? I wonder if there's a way to grab the last few items instead of having to update it in two spots. Can maybe look into it later.